ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/valuedisplay/ValueDisplayScreen.kt

```diff
@@ -0,0 +1,69 @@
+package com.example.android.wearable.composeadvanced.presentation.ui.valuedisplay
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.wear.compose.material.CompactChip
+import androidx.wear.compose.material.Text
+import com.example.android.wearable.composeadvanced.R
+
+/**
+ * Displays a value by using a Stepper or a Slider
+ */
+@Composable
+fun ValueDisplayScreen(
+    value: Int,
+    onClickStepper: () -> Unit,
+    onClickSlider: () -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(top = 10.dp)
+            .padding(horizontal = 10.dp),
+        verticalArrangement = Arrangement.Center
+    ) {
+        Text(
+            modifier = Modifier.fillMaxWidth(1f),
+            text = "$value",
```

Comment:
It will help to add a short text label before displaying actual value, e.g. 'previously selected value: ...'